src/frameworks/vite.json

```diff
@@ -9,7 +9,7 @@
   },
   "dev": {
     "command": "vite",
```

Comment:
@ascorbic, I thought about this one some more, and since this is the default if they have nothing configured, we can explicitly set it to `"command": "vite --port 5173",`? That way anyone who was using it previously won't run into any issues and new folks won't run into issues either.

If we were to use an environment variable, we'd have to use a vite config to consume it to set the port. The only problem with this is there's a disconnect between the command and the port fields, although this is a minor issue. We could also remove the `--port 5173` at some point since most people will probably have updated at some point.

Thoughts?

Reply:
I got this message with SvelteKit first changed and was not immediately aware that it couldn't find a port. It became clear eventually, but this message is stalled out. Is there a way to error out if it doesn't find one in a specific amount of time?

Reply:
I agree with @brittneypostma , erroring out after a specified amount of time would be useful so it removes the perception of 'hanging' and a useful error message can help the user troubleshoot a bit better with respect to the expected port for the framework

Reply:
Yeah, I think the timeout makes sense. Just wondering what makes sense in terms of time taking into account not everyone is running an M1. Maybe 30 seconds and make the message clearer? And if it fails to find it, we can add an error message like you suggested @ericapisani.

Reply:
> Maybe 30 seconds and make the message clearer?

I'm not sure if this is too aggressive, but I wonder if we would want to make the timeout a bit shorter (10-15 seconds) as I imagine that folks will ctrl/cmd+c the process sooner in the 30 second time frame 🤔 I'm not sure if we have any data to provide some guidance on this for us though so we can make a more informed decision

Reply:
I think 10-15 seconds may be too aggressive given that not everyone is running an M1. I think if we do 30 and make the message clearer about the port we should be good alongside the port change.

Reply:
Based on @seldo's comments in Slack, i.e. "People are overwhelmingly on version 3", I think it's safe to flick the switch to port 5173 and folks on older versions will get the port warning the CLI provides.